source/billing.txt

```diff
@@ -46,34 +47,36 @@ how much you pay each month:
 
 .. note:: Track Your Usage
    
-   You can track an application's usage for the current month from the
+   You can track an application's usage for the current day from the
    application dashboard in the App Services UI.
 
-Monthly Free Tier
-~~~~~~~~~~~~~~~~~
+Daily Free Tier
+~~~~~~~~~~~~~~~
 
-All App Services Apps in a MongoDB Atlas project share a single monthly free
-tier. All usage below the free tier thresholds in a given month is not billed.
-As soon as a project exceeds any monthly free tier threshold, App Services starts 
+All App Services Apps in a MongoDB Atlas project share a single daily free
+tier. All usage below the free tier thresholds in a given day is not billed.
+As soon as a project exceeds any daily free tier threshold, App Services starts 
 billing for additional usage of any kind for that project.
```

Comment:
[Important]

> App Services starts billing for additional **usage of any kind** for that project.

This bit isn't right anymore, is it? With the new pricing, we only charge for additional use of that specific type after they exceed the free tier.